### DISCUSSION TITLE
Support for MHO-C303

### DISCUSSION THREAD

**Discussion by cervalx** [pvvx/ATC_MiThermometer#443] [General]
This device is my favourite! It has a big E-Ink display that is visible from any angle, across the room; it has 2xAAA batteries so long battery life, it includes temp and humidity sensor.
I would really like to have custom BLE firmware and then Zigbee firmware.

@pvvx I am willing to order a device for you to take apart and develop on it, please let me know if you consider adding support for it.

**Comment by pvvx** [comment]
MHO-C303 is a 2020 production. And no one ever opened it to find out the chipset.

**Reply by cervalx** [reply]
It is also a bit difficult to open, does not have screws and comes out trough the front display. (which will ruin the estetics)
Would you still be interested to support it? From my side I can help to either open mine, or test a firmware on it.

**Reply by pvvx** [reply]
If TelinkMiFlasher.html writes " Not found Telink UUID service!" - this means that this device is not on a Telink chip. Then MHO-C303 is not interesting.

**Reply by cervalx** [reply]
Yes that is exactly the case. :( I understand, will buy a newer device.
